backend/apps/mentorship/management/commands/mentorship_update_comments.py

```diff
@@ -40,7 +40,22 @@ def process_mentorship_modules(self) -> None:
 
         self.stdout.write(self.style.SUCCESS("Starting mentorship issue processing job..."))
 
-        for module in published_modules:
+        modules_with_labels = published_modules.exclude(labels=[]).select_related("project")
+        modules_without_labels = published_modules.filter(labels=[])
+
+        if modules_without_labels.exists():
+            for module in modules_without_labels:
+                self.stdout.write(
+                    self.style.WARNING(f"Skipping. Module '{module.name}' has no labels.")
+                )
```

Comment:
I think there's still a better solution...

Do we really need to see this printed out? Do we care about this info at all? 🤔 This will require more computation and will slow down sync a little bit. Or a lot, depending on number of modules available.